src/libraries/System.Private.CoreLib/src/System/Collections/Generic/DebugViewDictionaryItem.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+namespace System.Collections.Generic
+{
+    /// <summary>
+    /// Defines a key/value pair for displaying an item of a dictionary by a debugger.
+    /// </summary>
+    [DebuggerDisplay("{Value}", Name = "[{Key}]")]
+    internal readonly struct DebugViewDictionaryItem<K, V>
```

Comment:
Generic naming conventions:
```suggestion
    internal readonly struct DebugViewDictionaryItem<TKey, TValue>
```

Reply:
Done.